src/__tests_/App.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from 'test-utils';
```

Comment:
with the use of `BrowserRouter` we no longer need to use the wrappers, so we can reference the imports directly from the testing-library

```suggestion
import { render, cleanup } from '@testing-library/react';
```

Reply:
Also correct, changed